Server/States/RunningState.cpp

```diff
@@ -48,6 +48,12 @@ void RunningState::ReadNetworkFunctions() {
         if (data.second.functionId == Replicated::RemoteServerCalls::GameLoaded) {
             AssignPlayer(data.first, GetPlayerObjectFromId(data.first));
         }
+        else if(data.second.functionId == Replicated::RemoteServerCalls::PlayerJump){
+            auto player = GetPlayerObjectFromId(data.first);
+            PlayerPhysComponent* playerPhysics;
+            player->TryGetComponent(playerPhysics);
```

Comment:
Use the bool from trygetcomponent to check if the component has been returned or not.